docs/DefiningLabels.md

```diff
@@ -0,0 +1,111 @@
+# Defining Labels
+
+Unless mentioned otherwise, **labels are applied to issues only** (not PRs). 
+
+## Label groups
+
+While GitHub does not have the concept of label groups, we can simulate label groups using systematic label naming. 
+i.e. labels having the same prefix are considered part of a label group.
+
+There are two types of label groups: 
+ 
+* Exclusive groups : No more than one label from the group can be applied to an issue
+* Non-exclusive groups:  Multiple labels from a group can be applied to an issue
+
+**Common label groups**:
+
+* exclusive : `s.` status, `p.` priority, `c.` category, `d.` difficulty, `e.` effort, 
+* non-exclusive:  `a-` aspect, `f-` feature, `t-` tech, 
+ 
+# Common Labels
+
+**Status (`s.`)**
+
+Status labels can be applied to PRs as well.
+
+* `s.OnHold` : The work on the PR has been put on hold pending some other event.
```

Comment:
You are right @pyokagan GitHub filter syntax hasn't caught up with the 'reviewers' features.  Even reviewable.io uses the `assignee` field to find reviewers instead of the `reviewers` field.